mtp_noms_ops/apps/security/forms/object_list.py

```diff
@@ -47,6 +49,86 @@ def get_api_request_params(self):
         return api_params
 
 
+class PrisonSelectorSearchFormMixin(forms.Form):
+    """
+    Mixin with prison fields for search V2.
+
+    prison_selector can be one of:
+    - all: all prisons
+    - mine: current user's prisons
+    - exact: for a specific prison
+
+    when prison_selector == exact a `prison` value must be specified
+    otherwise the form isn't valid.
+    when prison_selector != exact, any `prison` value is reset as not applicable
+    """
+    PRISON_SELECTOR_EXACT_PRISON_CHOICE_VALUE = 'exact'
+    PRISON_SELECTOR_ALL_PRISONS_CHOICE_VALUE = 'all'
+
+    prison_selector = forms.ChoiceField(
+        label=_('Prison selector'),
```

Comment:
does this visually appear anywhere? presumably error summary box? maybe it ought to be something like 'Choose a prison'… can wait for Jane to decide tho

Reply:
> presumably error summary box?

Yes and _Choose a prison_ is much better, thanks!